tests/integration/testdata/dump/002-skip-consumers/expected-no-skip-310.yaml

```diff
@@ -0,0 +1,67 @@
+_format_version: "3.0"
+consumer_groups:
+- name: basic
+  plugins:
+  - config:
+      compound_identifier: null
+      consumer_groups: null
+      dictionary_name: kong_rate_limiting_counters
+      disable_penalty: false
+      enforce_consumer_groups: false
+      error_code: 429
+      error_message: API rate limit exceeded
+      header_name: null
+      hide_client_headers: false
+      identifier: consumer
+      limit:
+      - 30000
+      lock_dictionary_name: kong_locks
+      namespace: basic
+      path: null
+      redis:
+        cluster_addresses: null
+        cluster_max_redirections: 5
+        cluster_nodes: null
+        connect_timeout: 2000
+        connection_is_proxied: false
+        database: 0
+        host: 127.0.0.1
+        keepalive_backlog: null
+        keepalive_pool_size: 256
+        password: null
+        port: 6379
+        read_timeout: 2000
+        redis_proxy_type: null
+        send_timeout: 2000
+        sentinel_addresses: null
+        sentinel_master: null
+        sentinel_nodes: null
+        sentinel_password: null
+        sentinel_role: null
+        sentinel_username: null
+        server_name: null
+        ssl: false
+        ssl_verify: false
+        timeout: 2000
+        username: null
+      retry_after_jitter_max: 0
+      strategy: local
+      sync_rate: null
```

Comment:
I see only this property is different from the config for v3.9, any reason why it is null?

Reply:
That's a question for the gateway team. 😅